app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -94,12 +99,19 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         setupThemeAndBrowsingMode(getModeFromIntentOrLastKnown(intent))
         setContentView(R.layout.activity_home)
         Performance.instrumentColdStartupToHomescreenTime(this)
-        setupToolbarAndNavigation()
 
         if (intent.getBooleanExtra(EXTRA_FINISH_ONBOARDING, false)) {
             FenixOnboarding(this).finish()
         }
 
+        externalSourceIntentProcessors.any {
```

Comment:
This line used to occur above the `if` block above. I haven't looked into what this line does but I think it should still go above the `if` block.